verifier/report_template.py

```diff
@@ -12,8 +12,22 @@ def __init__(self):
     <meta charset="UTF-8">
```

Comment:
Out of scope for now, but it would be better if you save big templates like this in a .html file and then imported them into Python, instead of having a big python string like this. It should help your tooling (e.g., syntax highlighting, auto format, etc).